src/components/ChainVerificationAnimation/Phase.tsx

```diff
@@ -35,21 +35,13 @@ export const PhaseColumn: React.FC<PhaseProps> = ({
     return isCompleted ? "border-green-500" : "border-amber-500";
   };
 
-  // Custom animation duration for DNSSEC Chain
+  // Calculate animation duration dynamically based on the number of steps
   const customStyle = isActive
     ? ({
-        "--animation-duration":
-          title === "DNSSEC Chain"
-            ? "6s"
-            : title === "X.509 Certificate Chain"
-              ? "2s"
-              : title === "CMS SignedData"
-                ? "1s"
-                : "1s",
+        "--animation-duration": `${Math.max(1, steps.length)}s`,
```

Comment:
This should be wrongly capping the animation duration at 1 second, but it isn't, so I wonder if this code is actually used?